.github/workflows/windows_conditional_compilation.yml

```diff
@@ -57,6 +57,7 @@ jobs:
     env:
       CMAKE_BUILD_TYPE: 'Release'
       CMAKE_GENERATOR: 'Ninja Multi-Config'
+      CMAKE_TOOLCHAIN_FILE: "${{ github.workspace }}\\openvino\\cmake\\toolchains\\mt.runtime.win32.toolchain.cmake"
```

Comment:
Please also add " /NODEFAULTLIB:libucrt.lib /DEFAULTLIB:ucrt.lib" to the toolchain